src/vendorcode/fsp/qemu/build.rs

```diff
@@ -43,37 +43,39 @@ fn main() -> std::io::Result<()> {
         // The input header we would like to generate bindings for.
         .header("src/wrapper.h")
         .clang_args(include_paths.iter().map(|include| format!("{}{}", "-I", include.display())))
+        .clang_args(&["-DEFIAPI=__attribute__((ms_abi))"])
         // Tell cargo to invalidate the built crate whenever any of the included header files
-        // changed.
+        // change.
         .parse_callbacks(Box::new(bindgen::CargoCallbacks))
         // Use core:: instead of std::
         .use_core()
+        .ctypes_prefix("cty")
         // Only generate types and constants.
         .with_codegen_config(bindgen::CodegenConfig::TYPES | bindgen::CodegenConfig::VARS)
         // Whitelist of types and constants to import.
```

Comment:
same but ... we have to repeat this everywhere? Is there some common crate we could have with this (later)?

Reply:
Yes, this can be put in a common crate (i think) (later).